roles/openshift_health_checker/openshift_checks/logging/logging.py

```diff
@@ -64,6 +64,17 @@ def not_running_pods(pods):
         ]
 
     @staticmethod
+    def running_pods(pods):
+        """Returns: list of pods in a running state"""
+        return [
+            pod for pod in pods
+            if any(
+                condition['type'] == 'Ready' and condition['status'] == 'True'
+                for condition in pod['status']['conditions']
```

Comment:
I believe that it can be in the `Running` state, but its endpoint not be reachable unless its `condition.status` is `true` https://github.com/kubernetes/kubernetes/issues/30785#issuecomment-240583200

We do the same sort of check for [not_running_pods](https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_health_checker/openshift_checks/logging/logging.py#L61)

but thinking about this a bit more, I think for this method, it should be enough to just check if the pod's `Phase == "Running"`